src/main/java/com/google/cloud/tools/gradle/appengine/standard/DevAppServerStopTask.java

```diff
@@ -41,10 +42,14 @@ public void setCloudSdkBuilderFactory(CloudSdkBuilderFactory cloudSdkBuilderFact
 
   /** Task entrypoint : Stop the dev appserver (get StopConfiguration from helper). */
   @TaskAction
-  public void stopAction() throws AppEngineException {
+  public void stopAction() throws CloudSdkNotFoundException {
     CloudSdk sdk = cloudSdkBuilderFactory.newBuilder(getLogger()).build();
 
     AppEngineDevServer server = serverHelper.getAppServer(sdk, runConfig);
-    server.stop(serverHelper.getStopConfiguration(runConfig));
+    try {
+      server.stop(serverHelper.getStopConfiguration(runConfig));
+    } catch (AppEngineException e) {
+      getLogger().warn("Failed to stop server: " + e.getMessage());
```

Comment:
Should this be at the error level?

Reply:
Yeah, I wasn't sure, but the more I look at it, the more "error" seems like the right level.